I.write-tests/01-greet-people/greet-people.test.js

```diff
@@ -1,7 +1,14 @@
-test('print list of names prefixed with Hello', function () {
-    // Arrange
+test("print list of names prefixed with Hello", function() {
+  // Arrange
+  //Given
+  var greetPeople = require("./greet-people");
+  var mentors = ["Irina", "Asleigh", "Etza"];
 
-    // Act
+  // Act
+  //When
+  var result = greetPeople(mentors);
 
-    // Assert
-})
\ No newline at end of file
+  // Assert
+  // Then
+  expect(result).toEqual("Hello IrinaAsleighEtza");
```

Comment:
This doesn't seem like quite what we want. I would think that we would at the very least want the three names to be separated! In TDD, it's good to set the expected value to what you want, see the test fail, and then correct the underlying code until the test passes.